frontend/apps/api_docs/v1/docs/finder_data_structures.apibp

```diff
@@ -85,10 +87,18 @@
                 + `poll_open_date`: `2017-05-04` (string) - Polling day for this ballot  (ISO 8601)
                 + `elected_role`: `Local Councillor` (string) - Name of the role the winner(s) of this election will assume
                 + metadata: (object, nullable) - Object containing information about special conditions for the user to be aware about (e.g: cancelled elections, voter id pilot). (details TBC)
-                + cancelled: `false` (boolean) - True if this ballot has been cancelled
-                + cancellation_reason: `null` (CancellationReason, nullable) - If a ballot has been cancelled, this key may hold a codified reason for that ballot's cancellation
+                + cancelled: `true` (boolean) - True if this ballot has been cancelled
+                + cancellation_reason: (CancellationReason, nullable) - If a ballot has been cancelled, this key may hold a codified reason for that ballot's cancellation
```

Comment:
This is a bit of an annoying limitation of API Blueprint/MSON/Drafter
If your data type is an enum, you can't say "make the value that goes in the docs example null" (previously this was inserting the string "null")
There's not really anything we can do about this. I made the example cancelled because `false` and a reason makes no sense